src/Strategy/NuGet/Nuspec.hs

```diff
@@ -66,11 +97,17 @@ data NuGetDependency = NuGetDependency
 instance FromXML Nuspec where
   parseElement el = do
     metadata     <- child "metadata" el
-    dependencies <- child "dependencies" metadata
-    Nuspec <$> children "group" dependencies
+    Nuspec <$> optional (child "dependencies" metadata >>= children "group") `defaultsTo` []
+           <*> optional (child "license" metadata)
+           <*> optional (child "licenseUrl" metadata)
+
+instance FromXML NuspecLicense where
+  parseElement el =
+    NuspecLicense <$> attr "type" el
+                  <*> content el
 
 instance FromXML Group where
-  parseElement el = Group <$> children "dependency" el
+  parseElement el = Group <$> optional (children "dependency" el) `defaultsTo` []
```

Comment:
[nit] I think this `defaultsTo` doesn't do anything -- `children` already returns an empty list if there aren't any